cmd/podman/containers/restore.go

```diff
@@ -116,26 +115,9 @@ func restore(cmd *cobra.Command, args []string) error {
 
 	if !exists.Value {
 		// Find out if this is an image
-		inspectOpts := entities.InspectOptions{}
-		imgData, _, err := registry.ImageEngine().Inspect(context.Background(), args, inspectOpts)
-		if err != nil {
-			return err
-		}
-
-		hostInfo, err := registry.ContainerEngine().Info(context.Background())
-		if err != nil {
-			return err
-		}
-
-		for i := range imgData {
-			restoreOptions.CheckpointImage = true
-			checkpointRuntimeName, found := imgData[i].Annotations[define.CheckpointAnnotationRuntimeName]
-			if !found {
-				return fmt.Errorf("image is not a checkpoint: %s", imgData[i].ID)
-			}
-			if hostInfo.Host.OCIRuntime.Name != checkpointRuntimeName {
-				return fmt.Errorf("container image \"%s\" requires runtime: \"%s\"", imgData[i].ID, checkpointRuntimeName)
-			}
+		restoreOptions.CheckpointImage, e = utils.IsCheckpointImage(context.Background(), args)
```

Comment:
nit Please use err rather then e.

Reply:
Thanks!

Reply:
Still valid.  I also find `e` confusing. `err` is the dominant convention in Go.